### DISCUSSION TITLE
It is not recommended to submit pnpm-lock

### DISCUSSION THREAD

**Discussion by vaynevayne** [heroui-inc/heroui#1329] [Ideas]
<img width="1026" alt="image" src="https://github.com/nextui-org/nextui/assets/124666577/e3fd2cf3-1a76-4515-ba24-b791514ea0c9">

Some packages, multiple packages for multiple operating systems

**Comment by jguddas** [comment]
> You should always commit the lockfile (pnpm-lock.yaml).

https://pnpm.io/git#lockfiles

**Reply by vaynevayne** [reply]
Okay, but the lock file locks the OS version, right? For example, if the lock file generated by windows is used, then the window package will not be installed on the mac, is that true? @jguddas
